tools/syz-cover/syz-cover.go

```diff
@@ -69,6 +66,17 @@ func main() {
 	if err != nil {
 		tool.Fail(err)
 	}
+	var pcs []uint64
+	if *flagCoverAll {
```

Comment:
Maybe just do it if no specific PCs are passed to `syz-cover` (i.e. `flag.Args()` is empty)?